test_venv.py

```diff
@@ -0,0 +1,73 @@
+# Copyright (c) 2012-2022 Esri R&D Center Zurich
```

Comment:
Nit: Should be 2023 ;)

Reply:
I think you missed the latest commit somehow where I fixed that